### PR TITLE
fix: Warning for multiple version

### PR DESCRIPTION
**Pull request by dipinknair**
use the ``warnings`` module to issue a warning instead of raising it as an exception.

### CODE REVIEW

File: src/ansys/mechanical/core/embedding/app.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def execute_script(self, script: str) -> typing.Any:
         if script_result.Error is not None:
             error_msg += f": {script_result.Error.Message}"
             raise Exception(error_msg)
-        return script_result.Value
+        return str(script_result.Value)
```
Comment (dipinknair) [inline src/ansys/mechanical/core/embedding/app.py:242]: @koubaa This is modified to be consistent with the remote session for future embedded rpc tests integration.

**Review by RobPasMue — APPROVED** [review]

File: src/ansys/mechanical/core/embedding/app.py (hunk 1 above)
Comment (koubaa) [reply src/ansys/mechanical/core/embedding/app.py:242]: @dipinknair I don't agree with this change. The server can do this, the embedding app must not

Comment (dipinknair) [reply src/ansys/mechanical/core/embedding/app.py:242]: Yeah, that is also true. Okay we can do that on server then.

Comment (dipinknair) [inline src/ansys/mechanical/core/embedding/app.py:242]:
```suggestion
        return script_result.Value
```

**Review by koubaa — APPROVED** [review]

**Pull request merged**